step.yml

```diff
@@ -0,0 +1,84 @@
+title: Appetize.io deploy
+summary: Deploy the app to Appetize.io
+description: |-
+  Deploy your app to Appetize.io and run it in your browser.
+  For app demos, training, customer support, development, testing, and more.
+website: https://github.com/bitrise-steplib/steps-appetize-io-deploy
+source_code_url: https://github.com/bitrise-steplib/steps-appetize-io-deploy
+support_url: https://github.com/bitrise-steplib/steps-appetize-io-deploy/issues
+
+project_type_tags:
+  - ios
+  - android
+  - xamarin
+  - react-native
+  - cordova
+  - ionic
+type_tags:
+  - deploy
+deps:
+  brew:
+  - name: go
+  apt_get:
+  - name: go
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-appetize-io-deploy
+
+inputs:
+  - app_path:
+    opts:
+      title: Application path
+      summary: The full path to the application file (.app or .apk)
```

Comment:
I would list all of the possibilities. .zip and .gz as well. Also I would split into cases like for android: apk, for ios: app, zip, ... also in desc.